server/routerlicious/packages/services-core/src/configuration.ts

```diff
@@ -7,6 +7,9 @@ import { IClientConfiguration, INackContent, NackErrorType } from "@fluidframewo
 
 // Deli lambda configuration
 export interface IDeliServerConfiguration {
+    // Enables nack messages logic
+    enableNackMessages: boolean;
```

Comment:
So setting this to false prevents the server from ever nacking?

Reply:
It prevents the server from sending the specific "Submit a summary before inserting additional operations" ones
Regular nacks will continue to work.

Reply:
Should this flag be renamed to reflect that it only enables / disables a specific type of nack message?

Reply:
The only r11s logic using the deli nackMessages system today is the scribe summary one. There might be more scenarios that use it in the future, so I don't think we should give it a specific name.
If we do get more scenarios, we may want to add some "scenario id" and a way to disable individual ones. But we can do that one we get there.

Reply:
sounds good.